frontend/src/component/notearea/NoteArea.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import "./NoteArea.css"
+import Searchbar from '../searchbar/Searchbar'
+import Noteinput from '../noteinput/Noteinput'
+import Notes from '../notes/Notes'
+
+function NoteArea() {
+  return (
+    <>
+    <div className="notes_area">
+    <Searchbar/>
+    <Noteinput/>
+    <Notes/>
+    </div>
```

Comment:
suggestion: use prettier for auto indentation

Reply:
Thanks for the feedback Rohit, I will use prettier.